subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/ConsoleJvmTestWorkerFunctionalTest.groovy

```diff
@@ -83,6 +83,27 @@ class ConsoleJvmTestWorkerFunctionalTest extends AbstractConsoleFunctionalSpec {
         gradleHandle.waitForFinish()
     }
 
+    def "shows abbreviated package when qualified test class is longer than 60 characters"() {
```

Comment:
Let's review and merge my PR that touches up the tests first. I made some improvements there: https://github.com/gradle/gradle/pull/2627.